Assets/Tests/InputSystem/CoreTests_Devices.cs

```diff
@@ -2674,7 +2674,7 @@ public void Devices_AnyKeyOnKeyboard_DoesNotReactToIMESelected()
     {
         var keyboard = InputSystem.AddDevice<Keyboard>();
 
-        InputSystem.QueueStateEvent(keyboard, new KeyboardState(Key.IMESelected));
+        InputSystem.QueueStateEvent(keyboard, new KeyboardState(IMESelected: true));
```

Comment:
Would be good to have a test that has an underlying struct/memory representation that looks like the previous one to prove that its backwards compatible?